### PR TITLE
Add manuSpecificBosch10 Cluster for BMCT-SLZ

### PR DESCRIPTION
**Pull request by slugzero**
This PR introduces the new manuSpecificBosch10 cluster for https://github.com/Koenkk/zigbee-herdsman-converters/pull/6785

### CODE REVIEW

File: src/zcl/definition/cluster.ts [hunk 1]
```diff
@@ -5427,6 +5427,20 @@ const Cluster: {
         },
         commandsResponse: {},
     },
+    manuSpecificBosch10: {
+        ID: 0xfca0,
+        manufacturerCode: ManufacturerCode.Bosch,
+        attributes: {
+            device_mode: {ID: 0x0000, type: DataType.enum8},
```
Comment (Koenkk) [inline src/zcl/definition/cluster.ts:5434]: Please make these camelCase, e.g. `deviceMode`

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee-herdsman-converters/pull/6785

File: src/zcl/definition/cluster.ts (hunk 1 above)
Comment (slugzero) [reply src/zcl/definition/cluster.ts]: done 👍

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**